### ISSUE TITLE
Test opengl backend for windows CI

### ISSUE THREAD

**Issue by ailzhang** [taichi-dev/taichi#3380]
this is blocked on a self-hosted windows GPU machine in CI.

**Comment by ailzhang** [comment]
Assigning to @chuandongyan who's looking into adding buildbot5 to CI. (Thanks!)

**Comment by ailzhang** [comment]
@frostming this is the issue that I mentioned about adding buildbot5 to CI. @chuandongyan will share access to buildbot5 to you :D. Thanks a lot!

**Comment by ailzhang** [comment]
This is not done yet :P - https://github.com/taichi-dev/taichi/pull/3852/files only enabled testing for cpu and cuda backends. @frostming would you mind sending another PR for opengl as well? Thanks a lot!
